lang/ko.json

```diff
@@ -0,0 +1,245 @@
+{
+	"appTitle": "CKFinder",
+
+	"common": {
+		"cancel": "취소",
+		"messageTitle": "정보",
+		"skip": "건너뛰기",
+		"rememberDecision": "내 선택 기억"
+	},
+
+	"dir": "ltr",
+	"LangCode": "ko",
+
+	"DateTime": "yyyy/m/d h:MM aa",
+	"DateAmPm": [
+		"AM",
+		"PM"
+	],
+
+	"FolderNew": "새로운 폴더 이름을 입력해 주십시오: ",
+	"FolderRename": "새로운 폴더 이름을 입력해 주십시오: ",
+	"FolderDelete": "정말 \"%1\" 폴더를 삭제하시겠습니까?",
+	"DestinationFolder": "목적",
+
+	"FileRename": "새로운 파일 이름을 입력해 주십시오	: ",
+	"FileRenameExt": "정말 파일 확장자를 변경하시겠습니까? 파일이 사용불능해 질 수 있습니다.",
+	"FileDelete": "정말 \"%1\" 파일을 삭제하시겠습니까?",
+	"FilesDelete": "정말 %1 파일들을 삭제하시겠습니까?",
+	"FilesLoading": {
+		"title": "로딩중...",
+		"text": "잠시만 기다려 주십시오. 해당 파일들이 로딩중입니다."
+	},
+	"FilesEmpty": {
+		"title": "폴더가 비어있습니다..",
+		"text": "<strong>업로드</strong> 버튼이나 파일을 이곳에 <strong>드레그&드롭</strong> 하십시오."
+	},
+	"FilterFilesEmpty": {
+		"title": "아무것도 찾지 못하였습니다.",
+		"text": "다른 검색어를 사용해 주십시오."
+	},
+
+	"OperationCompletedErrors": "오류와 함께 작업이 완료되었습니다.",
+	"MoveToolbarButton": "이동",
+	"CopyToolbarButton": "복사",
+	"MoveOneFileDialogTitle": "여기로 파일 이동...",
+	"CopyOneFileDialogTitle": "여기로 파일 복사...",
+	"MoveManyFilesDialogTitle": "여기로 {count} 개의 파일 이동...",
+	"CopyManyFilesDialogTitle": "여기로 {count} 개의 파일 복사...",
+	"MoveOneFileWait": "파일을 이동중입니다. 잠시만 기다려 주십시오.",
+	"CopyOneFileWait": "파일을 복사중입니다. 잠시만 기다려 주십시오.",
+	"MoveManyFilesWait": "%1 개의 파일을 이동중입니다. 잠시만 기다려 주십시오.",
+	"CopyManyFilesWait": "%1 개의 파일을 이동중입니다. 잠시만 기다려 주십시오.",
+	"MovedFilesNumber": "이동된 파일의 수: %1.",
+	"CopiedFilesNumber": "복사된 파일의 수: %1.",
+	"MoveOperation": "파일 이동중",
+	"CopyOperation": "파일 복사중",
+	"MoveFilesErrorTitle": "이동에 실패한 파일들:",
+	"CopyFilesErrorTitle": "복사에 실패한 파일들:",
+	"MoveCopyConfirmAction": "파일 작업 확인:",
+
+	"CopyDragDrop": "여기에 복사",
+	"MoveDragDrop": "여기로 이동",
+
+	"Upload": "업로드",
+	"Settings": "설정",
+	"Choose": "선택",
+	"ChooseScaledImage": "Choose Scaled",
```

Comment:
Any chance to add this missing string? This is the button label for an option that lets you choose/select a scaled (resized) image and send it to e.g. CKEditor instead of the original size. Thanks!